pants.toml

```diff
@@ -169,8 +169,8 @@ extra_env_vars = [
 ]
 
 [coverage-py]
-interpreter_constraints = [">=3.7,<3.10"]
-config = "build-support/coverage/coveragerc"
+interpreter_constraints = [">=3.9,<3.10"]
+config = "pyproject.toml"
```

Comment:
Do we actually need this now? I thought part of this was that we'd infer this.

Reply:
Won't be necessary in ~an hour :) This is all prework. Atm, we only warn you if we detect a relevant config file